core/src/main/java/org/fao/geonet/api/records/attachments/AbstractStore.java

```diff
@@ -206,6 +206,16 @@ public MetadataResource patchResourceStatus(final ServiceContext context, final
         return patchResourceStatus(context, metadataUuid, resourceId, metadataResourceVisibility, true);
     }
 
+    @Override
+    public void copyResources(ServiceContext context, String sourceUuid, String targetUuid, MetadataResourceVisibility metadataResourceVisibility, boolean sourceApproved, boolean targetApproved) throws Exception {
+        final List<MetadataResource> resources = getResources(context, sourceUuid, metadataResourceVisibility, null, true);
+        for (MetadataResource resource: resources) {
+            try (Store.ResourceHolder holder = getResource(context, sourceUuid, metadataResourceVisibility, resource.getFilename(), true)) {
```

Comment:
I believe "true" should be using variable sourceApproved

Reply:
fixed